portfolioLogic.php

```diff
@@ -2,3 +2,119 @@
 $db = new PDO('mysql:host=127.0.0.1; dbname=portfolioKyam', 'root');
 $db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
 
+/**
+ * Gets selected fields from database of first portfolio item
+ *
+ * @param $db to select from
+ * @return array of fields for item
+ */
+function getFirstPfItem(PDO $db) {
+    $query=$db->prepare("SELECT `title`,`description`,`imgRef`,`github`,`projURL`,`images`.`url`,`images`.`altText` 
+                         FROM `portfolioItems` 
+                         LEFT JOIN `images`
+                         ON `portfolioItems`.`imgRef`
+                         =`images`.`id`
+                         WHERE `portfolioItems`.`deleted` !=1 LIMIT 1;");
+    $query->execute();
+    $result= $query->fetchAll();
```

Comment:
function says you are getting first item and then you `fetchAll()`

Reply:
resolved